### ISSUE TITLE
ninja bombing needs converting to station beacons

### ISSUE THREAD

**Issue by Emisse** [space-wizards/space-station-14#23151]
the warp points are a bit jank and its better if its physical objects that exist
only thing that needs to be taken care of is to make sure objective is updated if its renamed or destroyed

**Comment by deltanedas** [comment]
new shitter tactic take all the ninja beacons and move them to some random thing in space / put them in arrivals

**Comment by Emisse** [comment]
can jsut give them a pinpointer to theirs anyway so it wont matter

**Comment by deltanedas** [comment]
still a bad thing, if ninja is told to bomb the armoury that should be what happens and is why its limited to a radius around the target. if they can just bomb anywhere the crew moves beacons to then might as well remove the limit and say bomb whatever you want

**Comment by Emisse** [comment]
hard coding areas into the game is bad

**Comment by keronshb** [comment]
> hard coding areas into the game is bad

Also giving players the ability to mess with antag objectives like that is bad.

If Ninja can't have a specific station area then the "bomb x area" should be reverted and then they just have C4 for the fun of it like the old days.

**Comment by IamVelcroboy** [comment]
Yeah, I don't know, making it a physical object that players can move and are aware of just seems...very gameable. Maybe it doesn't have to be warp points, but some way of designating specific rooms as bomb targets seems ideal.

**Comment by Emisse** [comment]
The objective should be changed in some way then, I am very against areas

**Comment by deltanedas** [comment]
it should either stay as mapped only or it should just be plain c4
